File: src/test/resources/fixtures/suppress/Overrides.java
class Overrides {
    enum CliOption { MAX_WORKERS, HOOK_TIMEOUT, TEARDOWN_TIMEOUT, TAGS_FILTER }
    static final String[] PROJECT_OVERRIDES = {"MAX_WORKERS", "TAGS_FILTER", "FILE_PARALLELISM"};
    int use(Config config) {
        int a = config.MAX_WORKERS;
        int b = config.HOOK_TIMEOUT;
        return a + b;
    }
}

File: src/test/resources/fixtures/suppress/overrides.ts
export type CliOption = 'maxWorkers' | 'hookTimeout' | 'teardownTimeout' | 'tagsFilter';

export const PROJECT_CLI_OVERRIDES = [
  'maxWorkers',
  // 'hookTimeout', // mirrorlint:ignore hookTimeout
  'teardownTimeout',
  'tagsFilter',
] as const;

export function apply(config: any) {
  const a = config.maxWorkers;
  const b = config.hookTimeout;
  const c = config["tagsFilter"];
  return a + b + c;
}
